tests/MRA.Jobs.Api/Application.UnitTests/TrainingModels/GetTrainingModelByIdQueryHandlerTests.cs

```diff
@@ -12,7 +12,7 @@ public override void Setup()
     {
```

Comment:
Can you add two integrations tests for these scenarios: 
IsApplied_ShouldBeFalse_WhenUserDidNotApply
IsApplied_ShouldBeTrue_WhenUserApplied